frontend/src/scenes/session-recordings/player/PlayerMetaLinks.tsx

```diff
@@ -29,6 +31,23 @@ export function PlayerMetaLinks(): JSX.Element {
         return Math.floor(playerTime / 1000)
     }
 
+    const commentInExistingNotebook = useCallback(
+        (notebookShortId) => {
+            const currentPlayerTime = getCurrentPlayerTime() * 1000
+            openNotebook(notebookShortId, NotebookTarget.Popover)
+            setTimeout(() => {
+                if (!nodeLogic) {
+                    throw new Error('wat')
+                }
+                // need time for the notebook to open and mount its logic
+                nodeLogic.actions.insertAfterLastNodeOfType(NotebookNodeType.ReplayTimestamp, [
```

Comment:
Might even need to get a little more clever here and make sure that the `ReplayTimestamp` relates the the `sessionRecordingId`. We already have some functions that rely on matching based on attributes: https://github.com/PostHog/posthog/blob/48515ccf8b7b3751d876ac32e9a81fff6d988002/frontend/src/scenes/notebooks/Notebook/Editor.tsx#L274C17-L287

Reply:
Ah, great timing thank you!

https://github.com/PostHog/posthog/pull/16955/commits/67f4c04839f47d1906004c17ae91f6177280498d

I've moved this "up" to the notebookLogic since that seems easier to mount and the nodeLogic is only used to get the insert position. But wanted to sense check I'm not missing a reason it was "down" in the nodeLogic

Reply:
Yeah, 👍  absolutely need to match node based on recording ID

Reply:
Nice! `hasMatchingNode` is exactly what I need 🙌